tests/integration/service_test.go

```diff
@@ -126,7 +126,7 @@ var _ = Describe("odoServiceE2e", func() {
 			)
 			ocArgs := []string{"get", "serviceinstance", "-o", "name", "-n", project}
 			helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
-				return strings.Contains(output, "dh-prometheus-apb")
+				return strings.Contains(output, "Provisioned")
```

Comment:
The command is outputting just service name, so this is not going to work.

Reply:
Ahh, Will fix it

Reply:
Done